.devcontainer/Dockerfile

```diff
@@ -84,17 +108,8 @@ ENV CXX=${USE_CLANG:+"clang++"}
 ENV CC=${CC:-"gcc"}
 ENV CXX=${CXX:-"g++"}
 
-# By default, anything you run in Docker is done as superuser.
-# Conan runs some install commands as superuser, and will prepend `sudo` to
-# these commands, unless `CONAN_SYSREQUIRES_SUDO=0` is in your env variables.
-ENV CONAN_SYSREQUIRES_SUDO 0
-# Some packages request that Conan use the system package manager to install
-# a few dependencies. This flag allows Conan to proceed with these installations;
-# leaving this flag undefined can cause some installation failures.
-ENV CONAN_SYSREQUIRES_MODE enabled
-
 # Include project
-ADD . /starter_project
-WORKDIR /starter_project
+#ADD . /workspaces/cpp_starter_project
```

Comment:
@ddalcino The more I see this copy the less best practice it seems to me. This is because if you make changes in the code inside the container they will be lost when you delete that container. The way to go here is to create a volume as described in the readme file (if you don't use VSCode that does that automatically). So, I suggest to remove that entirely to avoid loosing code changes. Also, it takes time when you build the container. What do you think?

Reply:
I think it depends on how you’re using the Dockerfile. If you’re developing locally, then it doesn’t make any sense to copy the source folder. If it’s part of your CI/CD pipeline, it might be easier to copy the source code than to mount a directory. Everyone’s needs are different.

Reply:
> 

Shouldn't the CI clone the repo and check it out to the correct branch before everything else? Well I am not qualified by any means when it comes to the CI jobs.. So should I un-comment those two lines?

Reply:
Those lines should definitely be commented out, if not removed entirely. I think that the Dockerfile should work properly with VSCode out-of-the-box. If someone is using the Dockerfile in their CI, they will need to make several modifications to get everything working the way they want it to. I don't think that we can predict what those modifications should be.

Honestly, I do not know if we should keep these two lines as comments, or if we should remove them. As comments, they do not bother me. I think that what you have for this file is just fine.